src/main/kotlin/com/fasterxml/jackson/module/kotlin/internal/KotlinValueInstantiator.kt

```diff
@@ -24,12 +27,12 @@ import kotlin.reflect.jvm.isAccessible
 import kotlin.reflect.jvm.javaType
 
 internal class KotlinValueInstantiator(
```

Comment:
Note that Kotlin's `internal` keyword is meaningless because we only have a single (compilation) module.

https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces